### PR TITLE
Admin QoL: Deletes stickybans, tidies mob VV menu

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Removes the long defunct sticky ban system. (We use a much more advanced ban evasion detection method)
Removes some random tat from mob VV dropdown menu that is also present on the PP

## Why It's Good For The Game
Less junk and random tat for admins to swim through. Literally nobody uses these in the dropdown on mob VV panel. Everyone uses the PP for these functions.
We (apparently) had only 6 sticky bans total in the history of para. (vs approx 64k "normal" bans)

## Testing
Loaded up, stickyban menu gone, VV dropdown tidier.

## Changelog
:cl:
del: Removed duplicate entries from mob VV dropdown
del: Removed defunct stickyban system
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1915,20 +1915,6 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
...
-	.["Set Species"] = "?_src_=vars;setspecies=[UID()]"
-	.["Copy Outfit"] = "?_src_=vars;copyoutfit=[UID()]"
-	.["Make AI"] = "?_src_=vars;makeai=[UID()]"
-	.["Make cyborg"] = "?_src_=vars;makerobot=[UID()]"
-	.["Make monkey"] = "?_src_=vars;makemonkey=[UID()]"
-	.["Make alien"] = "?_src_=vars;makealien=[UID()]"
-	.["Make slime"] = "?_src_=vars;makeslime=[UID()]"
-	.["Make superhero"] = "?_src_=vars;makesuper=[UID()]"
-	. += "---"
-
```
Comment (AffectedArc07) [inline code/modules/mob/living/carbon/human/human.dm:1931]: Remove the callbacks for these maybe?

Comment (S34NW) [reply code/modules/mob/living/carbon/human/human.dm:1931]: they are used in PP

Comment (AffectedArc07) [reply code/modules/mob/living/carbon/human/human.dm:1931]: > they are used in PP

Not exactly.
image

Comment (S34NW) [reply code/modules/mob/living/carbon/human/human.dm:1931]: Oh they are in a different topic? Well damn, alright I'll see what I can do

**Review by AffectedArc07 — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**